buildpacks/nodejs-npm-install/src/npm.rs

```diff
@@ -0,0 +1,69 @@
+use commons::fun_run::CmdError;
+use libcnb::Env;
+use std::path::PathBuf;
+use std::process::Command;
+
+pub(crate) struct SetCacheConfig<'a> {
+    pub(crate) env: &'a Env,
+    pub(crate) cache_dir: PathBuf,
+}
+
+impl<'a> From<SetCacheConfig<'a>> for Command {
```

Comment:
Why not use a simple function like `fn set_cache_config(env, cache_dir) -> Command` rather than the `From` `impl`? This conversion just feels a little indirect and a surprising primary API.

Reply:
Maybe.  I was borrowing this style from how @Malax and @edmorley implemented commands for tools like [pack](https://github.com/heroku/libcnb.rs/blob/main/libcnb-test/src/pack.rs#L85) and [docker](https://github.com/heroku/libcnb.rs/blob/main/libcnb-test/src/docker.rs#L69) in `libcnb.rs` and, though it also seemed indirect to me at first, I found it useful for separating the command and arguments from how it's invoked.